leasing/management/commands/attach_areas.py

```diff
@@ -76,11 +86,11 @@ def handle(self, *args, **options):  # noqa: C901 TODO
                         .exclude(geometry__touches=area.geometry)
                     )
                 except InternalError as e:
-                    self.stdout.write(str(e))
+                    log.exception(str(e))
```

Comment:
Don't generate the error message from the exception object `e`, since info about the exception is already present in the message generated by the `exception` method. Rather use some static string like `"Failed to get intersected areas"`. This string can then be used to search for this code location, when you encounter that string in the logs, or the other way around to see how many times that error happens by searching it from the logs. I wouldn't even know how to find these `self.stdout.write(str(e))` messages from the current logs, since their content is so dynamic.

Here's an example how the `exception` method behaves:
```
>>> import logging
>>> logging.basicConfig(level=logging.INFO)
>>> LOG = logging.getLogger('test')
>>> try:
...     raise IndexError("Index finger missing! Ouch!")
... except Exception:
...     LOG.exception("Finger search failed")
... 
ERROR:test:Finger search failed
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
IndexError: Index finger missing! Ouch!
```

Reply:
Changed to static string